cv-data-service-library/src/main/java/com/trihydro/library/helpers/DbInteractions.java

```diff
@@ -17,68 +17,87 @@
 @Component
 public class DbInteractions {
     private static HikariDataSource dataSource;
-    private static HikariDataSource dataSourceCountyRoads;
 
     protected DbInteractionsProps dbConfig;
     protected Utility utility;
-    protected EmailHelper emailHelper;
+    private EmailHelper emailHelper;
 
     @Autowired
     public void InjectDependencies(DbInteractionsProps props, Utility _utility, EmailHelper _emailHelper) {
         dbConfig = props;
         utility = _utility;
         emailHelper = _emailHelper;
-        utility.logWithDate("A new DbInteractions instance has been created.", this.getClass());
-        validateDbConfig();
-    }
+        utility.logWithDate("DbInteractions: Injecting dependencies");
+        initDataSources();
```

Comment:
I would strongly recommend keeping the https://dev.azure.com/trihydro/CV/_git/WyoCV/pullrequest/8282 code changes separate from this changeset. Once we get this PR merged, it will be much easier to apply and review the changes from https://dev.azure.com/trihydro/CV/_git/WyoCV/pullrequest/8282

Reply:
That sounds reasonable to me! Opened an issue here: https://github.com/Trihydro/TIM-Manager/issues/5